aas-gui/Frontend/aas-web-gui/.prettierrc

```
@@ -0,0 +1,11 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 2,
```

This setting would fit better to the current state of the project:
```suggestion
    "tabWidth": 4,
```